### PR TITLE
Player: Implement PlayerContinuousLongJump

### PR DESCRIPTION
**Pull request by MonsterDruide1**
Seems like there is also a counter for the number of continuous long jumps in the game, which helps in selecting a different animation each time - resulting in a total of 8 possible, selectable values.

However, the `PlayerConst` limits this to a maximum of 3 values by default, resulting in the commonly-known animations. By overriding this, we can view some more animations that have not been used in the final game.

Looking at `PlayerAnimation.szs`, we can see `JumpBroad`, `JumpBroad2`, ..., `JumpBroad9` are existing animations. Which means that even when overriding the PlayerConst limit, there is still one animation left in the game files that cannot be viewed in-game *at all* without `exefs` mods.

To summarize again: Nintendo limited us to see 3/9 animations by `PlayerConst`, and 8/9 by the array of hardcoded, supported LongJump-Animation names.

- - -
This change is [Reviewable](https://reviewable.io/reviews/MonsterDruide1/OdysseyDecomp/54)

### CODE REVIEW

**Review by tetraxile — COMMENTED** [review]
pretty interesting!
:lgtm:

> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/MonsterDruide1/OdysseyDecomp/54#-:-NzKkSFG2-xREpfN_R24:b2287e0)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @MonsterDruide1)
